spec/serializers/spree_signifyd/signifyd_case_serializer_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
```

Comment:
I think that this spec should now go into the order_serialize_spec. The original plan had been to rename all of the serializers and replace then with jbuilder, but I no longer have the heart to do that.

I don't believe that AMS was a good choice. We are not *serializing* our models for SIGNIFYD, we are *using* our models to construct their concepts. It just happens that many of their models share commonalities with ours, but where we bump into the edges things get really gross really fast.

The reason the order_serializer is full of constructed hashes is because it is trying to contort an Order into the various aspects of a Purchase (whatever SIGNIFYD decides that is). This also shows up when we do gymnastics in the user_serializer

```
def lastOrderId
  completed_orders.order("completed_at DESC").second.try(:number)
end
```

That being said, since our models are actually very close to their models, I think AMS and the status quo is fine for now. I'll move this spec into the other file.